mongo_connector/connector.py

```diff
@@ -704,19 +713,53 @@ def apply_namespaces(option, cli_values):
             raise errors.InvalidConfiguration(
                 "Namespace set should not contain any duplicates.")
 
+        ex_ns_set = option.value['exclude']
+        if len(ex_ns_set) != len(set(ex_ns_set)):
+            raise errors.InvalidConfiguration(
+                "Exclude namespace set should not contain any duplicates.")
+
+        # not allow to exist both 'include' and 'exclude'
+        if ns_set and ex_ns_set:
+            raise errors.InvalidConfiguration(
+                "Include and exclude namespace sets"
+                " are not allowed to exist both.")
+
+        # validate 'include' format
+        for ns in ns_set:
+            if ns.count("*") > 1:
+                raise errors.InvalidConfiguration(
+                    "Namespace set should be plain text "
+                    "e.g. foo.bar or only contains one wildcard, e.g. foo.* .")
+
+        # validate 'exclude' format
+        for ens in ex_ns_set:
+            if ens.count("*") > 1:
+                raise errors.InvalidConfiguration(
+                    "Exclude namespace set should be plain text "
+                    "e.g. foo.bar or only contains one wildcard, e.g. foo.* .")
+
         dest_mapping = option.value['mapping']
         if len(dest_mapping) != len(set(dest_mapping.values())):
             raise errors.InvalidConfiguration(
                 "Destination namespaces set should not"
                 " contain any duplicates.")
 
+        for key, value in dest_mapping.items():
+            if key.count("*") > 1 or value.count("*") > 1:
+                raise errors.InvalidConfiguration(
+                    "Only one * character is allowed in the namespaces!")
```

Comment:
Change to "The namespace mapping source and destination cannot contain more than one '*' character.